### DISCUSSION TITLE
How to run streetmerch after restarting computer

### DISCUSSION THREAD

**Discussion by Limpyy** [jef/streetmerchant#1489] [Help] [answered]
Once I restart my computer I cant seem to run streetmerch again on the command prompt, what am I supposed to type in?

**Comment by jef** [comment, marked as answer]
You should navigate to where you installed streetmerchant. You can right click in the explorer and open a command prompt there. Run `npm run start` within there.

**Comment by Limpyy** [comment]
It worked, Thank you!

**Reply by jef** [reply]
You're welcome!

**Comment by FlyingToto** [comment]
BTW,  if you are on windows, you can use "task scheduler" to run the task automatically upon reboot (or even restart it every 2 h...)

- click the windows menu, type "task scheduler"
- create a new task, call it whatever you want "looking for the holy grail"
- command to run: npm
- arguments: run start
- path to run: the path where you installed streetmerchant
- trigger: on restart
